modules/appeals_api/config/schemas/shared/v1/address.json

```diff
@@ -22,7 +21,7 @@
             "WI", "WY"
           ]
         },
-        "countryCodeISO2": { "type": "string", "pattern": "^[A-Z]{2}$", "minLength": 2, "maxLength": 2 },
+        "countryCodeISO2": { "type": "string", "pattern": "^[A-Z]{2}$" },
```

Comment:
initially changed this to fix test-breaking logic in `schema_after_property_validation`, but also realized that the regex pattern makes the min/max length properties redundant

Reply:
Min/max properties may be redundant based on the pattern but we had specifically added them at the direction of the API Styleguide, so let's keep them in place. It sounds like there may be some more work being done on the styleguide so we could reach out to see if this is something that's going to be necessary in the next revision.